html/components/button.stories.js

```diff
@@ -25,6 +25,10 @@ always use a \`<button>\` element.
 must include a \`title=””\` attribute.
 - If a Button only includes an Icon with no text,
 alternative text must be provided.
+- If the Button has a spinner, the \`aria-label\`
+will be set to 'Loading' unless you supply a custom
+string value to the \`data-sprk-spinner-aria-label\`
+attribute.
```

Comment:
I would add something here that makes it clear this change only happens in the click event on the button, not when it is initialized.

Reply:
ohh good idea!